js/actions/index.ts

```diff
@@ -74,7 +93,12 @@ function _receivePortalData(db: firebase.firestore.Firestore,
     type: RECEIVE_PORTAL_DATA,
     response: rawPortalData
   });
-  let resourceUrl = rawPortalData.offering.activity_url.toLowerCase();
+  let resourceUrl;
+  if ((rawPortalData.offering.activity_url).includes("activity=")) {
```

Comment:
This is a kind of hack to support reporting on activity player based external activities. In those cases the url will look something like: `https://activity-player.concord.org?activity=https%3A%2F%2Fauthoring.concord.org%2Factivity%2F123`

A comment to this effect should be added to this code.

Also the parsing of the URL is fragile. It'd be better to use the queryString library to parse the url and pull out the activity parameter.

Reply:
I think this code can stay as it is. We'll likely be refactoring it when cleaning up the changes needed to run the report directly from the portal.